src/lib/tracker.ts

```diff
@@ -277,6 +277,8 @@ export default function Plausible(
     }
   ) => {
     function trackClick(this: HTMLAnchorElement, event: MouseEvent) {
+      if (event.ctrlKey) return; // Don't send event on Ctrl + Click, should behave like middle-mouse-click.
```

Comment:
No, it absolutely can. In my tests, middle click just never sent the event and I wanted ctrl + click to behave the same as middle click. Out of the two options I went with don't send (and didn't think much of it) but as I mentioned in [my last comment on #12](https://github.com/plausible/plausible-tracker/issues/12#issuecomment-977609255), this would lead to a discrepancy with the script in the main repo.

I'll update the PR. Should I just use the logic from the main script for middle click handling (I've got no experience there)? https://github.com/plausible/analytics/blob/7e93500834cc17000c0d2887b1ea9cd936828409/tracker/src/plausible.js#L84

Reply:
Ah OK I see. I think it's better to attempt to send it in  every case possible where one would expect the click to be tracked.

Yes, you can take the middle click behaviour from the main repo.

You're right the reason we don't use the same codebase is due to size. For the main script we have optimized it for UglifyJS so it can be compressed down as much as possible. However, for the NPM module we wanted to have a more 'proper' SDK instead of a single-file script.

This does introduce issues with keeping them in sync. It's a manual job at the moment. After this PR is done, the `window.top` and `window.parent` behaviour should be ported to the main repo as well

Reply:
I will as soon as I get more time, thanks.

Regarding the main script, have you tried building an optimized version of the script with the npm module using tree shaking, webpack (or similar) and UglifyJs? I'd imagine it won't be quite as small but you could probably get close, no?

Reply:
I haven't tried. The history is that the script in the main repo was written first and this NPM module was contributed later by an external contributor.